pgoapi/pgoapi.py

```diff
@@ -178,6 +183,14 @@ def __init__(self, config):
         self.should_catch_pokemon = config.get("CAPTURE", {}).get("CATCH_POKEMON", True)
         self.max_catch_attempts = config.get("CAPTURE", {}).get("MAX_CATCH_ATTEMPTS", 10)
 
+        self.new_forts = []
+        self.cache_filename = './cache/cache ' + str(config["location"]) + str(self.STAY_WITHIN_PROXIMITY)
```

Comment:
@destiny117 hmmm, copied the syntax from the pgoapi and pokecli https://github.com/j-e-k/poketrainer/blob/master/pokecli.py#L123

Should error there before it gets to my code :P